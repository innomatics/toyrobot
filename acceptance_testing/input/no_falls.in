PLACE 0,0,SOUTH
MOVE
REPORT
RIGHT
RIGHT
MOVE
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT
RIGHT
MOVE
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT
RIGHT
MOVE
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT
RIGHT
MOVE
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT