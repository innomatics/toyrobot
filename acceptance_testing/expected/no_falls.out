0,0,SOUTH
0,4,NORTH
4,4,EAST
4,0,SOUTH
0,0,WEST
